style.css

```diff
@@ -1 +1,304 @@
-/*  Add your styling here */
+@import url(https://fonts.googleapis.com/css2?family=Dancing+Script:wght@700&family=Roboto:wght@100&display=swap);
+
```

Comment:
Consider putting your CSS files to a separate folder. In real project you will most likely use multiple CSS files anyway.